### ISSUE TITLE
[TF2] [Art] Reverse the team coloring for the Unarmed Combat

### ISSUE THREAD

**Issue by Chickenman456** [ValveSoftware/Source-1-Games#5321]
The Unarmed Combat was originally designed to have scout wielding the opposite team's spy arm.

This was an intentional decision, as seen in the original workshop submission - https://steamcommunity.com/sharedfiles/filedetails/?id=1813

This was originally implemented as intended, but was "fixed' during the Pyromania update.

![image](https://github.com/ValveSoftware/Source-1-Games/assets/87006112/03219598-9462-43e9-9601-326bfaf06e86)

**Comment by Wosheereal** [comment]
This would be done by swaping the $basetexture filepaths with both teams. Blu for red and Red for blu if valve decides to replace the original look instead of chickenman's proposal.

**Comment by JoriKos** [comment]
This was most likely an intentional decision to not mess with the team colouring. Considering how much people focus on that all of a sudden, probably not the best idea to implement this now. Add a strange variant though :^)

**Comment by Chickenman456** [comment]
> This was most likely an intentional decision to not mess with the team colouring. Considering how much people focus on that all of a sudden, probably not the best idea to implement this now. Add a strange variant though :^)

I agree that’s probably the reason- but I don’t think it’s a super big deal considering we now have the warpaint system, which includes non-team colored blue paints.

**Comment by gabrielwoj** [comment]
The Bat Outta Hell has a similar thing, the Scout and Soldier styles use team-opposite colors on the headset and helmet, respectively. Although it's a lot less noticeable than the Unarmed Combat.
